src/ComposerIntegration/InstallAndBuildProcess.php

```diff
@@ -43,6 +48,27 @@ public function __invoke(
             $downloadedPackage->extractedSourcePath,
         ));
 
+        if (file_exists(rtrim($downloadedPackage->extractedSourcePath, '/') . '/.gitmodules')) {
+            try {
+                $output->writeln(sprintf(
+                    '<info>Found .gitmodules file in</info> %s<info>, fetching submodules...</info>',
+                    $downloadedPackage->extractedSourcePath,
+                ));
+
+                $git              = GitBinaryPath::fromGitBinaryPath('/opt/homebrew/bin/git');
```

Comment:
This path is specific to macOS package manager, it won't work on other systems. 

I'm not even sure it is needed to specify git path as it should be in `$PATH` of the user

Reply:
Ah yes indeed, it was for test purposes to validate the implementation. Thanks for spotting it 👍